src/Runner.jl

```diff
@@ -420,6 +440,9 @@ function generate_compiler_wrappers!(platform::AbstractPlatform; bin_path::Abstr
         if lock_microarchitecture
             append!(flags, get_march_flags(arch(p), march(p), "gcc"))
         end
+        if libc(platform) == "musl"
```

Comment:
This should be needed only for GCC 4.8.5, right?  You can check the version number like in https://github.com/JuliaPackaging/BinaryBuilderBase.jl/blob/c3a40125ecdb2869c8d0355389f88122fd89d1e8/src/Runner.jl#L429-L430

Reply:
Added `gcc_version == v"4.8.5"` check

Reply:
@giordano OK now?

Reply:
`gcc_version` is undefined there, or am I missing something?

Reply:
Argh, indeed. I've now added
```julia
        gcc_version, llvm_version = select_compiler_versions(p, compilers)
```

Reply:
Mumble mumble mumble... this also means this line is untested, which is very weird to be honest :confused:

Reply:
Nevermind, it didn't error because there is a function called `gcc_version`: https://github.com/JuliaPackaging/BinaryBuilderBase.jl/blob/c3a40125ecdb2869c8d0355389f88122fd89d1e8/src/Rootfs.jl#L433 :grimacing:

Reply:
good that you paid close attention then 😅